easybuild/main.py

```diff
@@ -357,6 +373,12 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
                     log=_log, opt_parser=eb_go.parser, exit_on_error=not testing)
     _log.debug("Paths: %s", paths)
 
+    # if we are only copying ec's for a specific PR we can do that now
+    if options.copy_ec and options.from_pr:
+        determined_paths = [path[0] for path in paths]
+        copy_ecs_to_target(determined_paths, target_path)
```

Comment:
The change to use an absolute path for `target_path` was required because at this point, if we use `--from-pr`, we are inside a temporary directory so a relative path points to the wrong place.

Reply:
I agree that this behaviour is probably a bit of a bug